.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build And Formatting CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+      
+    - name: Install Dependencies
+      run: npm i
+      
```

Comment:
Add a step for checking tests too.
`npm run test`

Reply:
I was thinking we could have another workflow for test exclusively or maybe extend this one at some later point when we have tests (?)
Since we don't have tests right now

Reply:
I see a test here: https://github.com/glugmvit/gwoc/blob/master/src/App.test.js
One test is enough for the script to run. No harm in adding the check, just a good practice.
You can add the tests check in a different workflow if you like, that's up to you. But if we have a test script, it'd be pointless not to use it somewhere in CI checks.